openapi/components/schemas/Files/File.yaml

```
@@ -1,9 +1,11 @@
 type: object
 properties:
   id:
+    type: string
+    description: File ID.
```

nitpick: we should use this format
```suggestion
    description: ID of the file.
```